packages/dart/sshnp_flutter/lib/src/presentation/widgets/settings_actions/settings_privacy_policy_action.dart

```diff
@@ -13,8 +13,7 @@ class SettingsPrivacyPolicyAction extends StatelessWidget {
       icon: Icons.account_balance_wallet_outlined,
       title: strings.privacyPolicy,
       onTap: () async {
-        final Uri url =
-            Uri.parse('https://atsign.com/apps/atdatabrowser-privacy-policy/');
+        final Uri url = Uri.parse('https://atsign.com/apps/atdatabrowser-privacy-policy/');
```

Comment:
Wrong app's privacy policy linked here

Reply:
fixed. We're using atsign privacy policy for now until we finalise noports privacy policy.